src/Services/MeilisearchService.php

```diff
@@ -155,7 +155,7 @@ public function search(
     ): array {
         $this->assertIsSearchable($className);
 
-        $ids = $this->engine->search($query, $this->searchableAs($className), $searchParams);
+        $ids = $this->engine->search($query, $this->searchableAs($className), $searchParams + ['limit' => $this->configuration['nbResults']]);
```

Comment:
Shouldn't we use an `array_merge` or something like that?

Reply:
No need :) test proves that:)